MarvelApiClient/src/test/java/com/karumi/MarvelApiClient/CharacterApiClientTest.java

```diff
@@ -0,0 +1,251 @@
+/*
+ *   Copyright (C) 2015 Karumi.
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package com.karumi.marvelapiclient;
+
+import com.karumi.marvelapiclient.model.CharacterDto;
+import com.karumi.marvelapiclient.model.CharactersDto;
+import com.karumi.marvelapiclient.model.CharactersQuery;
+import com.karumi.marvelapiclient.model.ComicResource;
+import com.karumi.marvelapiclient.model.MarvelImage;
+import com.karumi.marvelapiclient.model.MarvelResources;
+import com.karumi.marvelapiclient.model.MarvelResponse;
+import com.karumi.marvelapiclient.model.MarvelUrl;
+import com.karumi.marvelapiclient.model.StoryResource;
+import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.List;
+import java.util.TimeZone;
+import org.junit.Test;
+import retrofit.GsonConverterFactory;
+import retrofit.Retrofit;
+
+import static org.junit.Assert.assertEquals;
+
+public class CharacterApiClientTest extends ApiClientTest {
+
+  private static final String ANY_PUBLIC_KEY = "1234";
+  private static final String ANY_PRIVATE_KEY = "abcd";
+  private static final String ANY_URL = "http://fake.marvel.com/";
+  private static final int ANY_OFFSET = 1;
+  private static final int ANY_LIMIT = 10;
+  private static final int INVALID_LIMIT = 0;
+  private static final String ANY_NAME = "Spider-man";
+  private static final String ANY_START_NAME = "Spider";
+  private static final int ANY_COMIC_1 = 1;
+  private static final int ANY_COMIC_2 = 2;
+  private static final int ANY_SERIE = 1;
+  private static final int ANY_STORY = 1;
+  private static final int ANY_EVENT_1 = 1;
+  private static final int ANY_EVENT_2 = 2;
+  private static final String ANY_MODIFIED_SINCE = "2015-01-09T22:10:45-0800";
+  private static final String ORDER_NAME_DESCEDANT_VALUE = "-name";
+  private static final String INVALID_CHARACTER_ID = "";
+  private static final String ANY_NOT_FOUND_ID = "1234";
+  private static final String THREED_MAN_CHARACTER_ID = "123456";
+
+  @Test public void shouldReturnAValidUrlWhenICallToGetAllWithValidOffsetAndLimit()
+      throws Exception {
+    CharacterApiClient characterApiClient = givenCharacterApiClient();
+    enqueueMockResponse(200);
```

Comment:
We should create a method ``enqueueMockResponse()`` using no params and a default 200 response. Actually, the default ``MockResponse`` used by mock web server is a 200 http status code.